src/errors/Messages.js

```diff
@@ -64,7 +64,6 @@ const Messages = {
 
   VOICE_NOT_STAGE_CHANNEL: 'You are only allowed to do this in stage channels.',
 
-  VOICE_STATE_UNCACHED_MEMBER: 'The member of this voice state is uncached.',
```

Comment:
The removal of this message makes the PR semver-major.

Reply:
Why though?

Reply:
On a second thought, it's not, messages aren't exported and they're only semver-major when the message changes, sorry 😅